# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ppracer/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur ppracer-0.3.1-orig/src/ppgltk/FT/FTTextureGlyph.h ppracer-0.3.1/src/ppgltk/FT/FTTextureGlyph.h
--- ppracer-0.3.1-orig/src/ppgltk/FT/FTTextureGlyph.h	2005-02-18 10:16:04.000000000 +0000
+++ ppracer-0.3.1/src/ppgltk/FT/FTTextureGlyph.h	2006-04-03 21:21:24.431998000 +0000
@@ -52,7 +52,7 @@
          * Reset the currently active texture to zero to get into a known state before
          * drawing a string. This is to get round possible threading issues.
          */
-        static void FTTextureGlyph::ResetActiveTexture(){ activeTextureID = 0;}
+        static void ResetActiveTexture(){ activeTextureID = 0;}
         
     private:
         /**
